### ISSUE TITLE
Allow reuse of cert key (HPKP)

### ISSUE THREAD

**Issue by kuba** [kuba/simp_le#1]
`--reuse_key`?

**Comment by 8h2a** [comment]
Or give a public key via argument?
`--pub_key keyfile`

But this may be a stupid idea since I'm not sure yet how exactly this application works. (Is there a documentation that describes what exactly this application does, besides the source code?)

**Comment by siccegge** [comment]
Also necessary to make this of any use with DANE btw.

You really don't want to have this copy new fingerprints to your nameserver as well

**Comment by kuba** [comment]
This was fixed in 1cd1690. Let me know if you have any problems, questions :)

**Comment by kuba** [comment]
As of e6fb0423a260250bf496da49be5030c250233431, the following happens

```
(venv)jakub@iks:~/dev/letsencrypt/simp_le$ ls *.pem                                                             
(venv)jakub@iks:~/dev/letsencrypt/simp_le$ staging_simp_le -d kuba.us.to:public_html -f key.pem -f fullchain.pem
Existing data is missing some components. Are you using the same plugins as previously?                         

Debugging tips: -v improves output verbosity. Help is available under --help.                                   
```

Instead, we would like to be able to use already generated key before any certificate is created.

**Comment by kuba** [comment]
This should have been fixed in 0902658.
